client/me/purchases/manage-purchase/index.jsx

```diff
@@ -208,9 +208,9 @@ class ManagePurchase extends Component {
 	};
 
 	handleRenewMultiplePurchases = ( purchases ) => {
-		const { siteSlug, redirectTo } = this.props;
+		const { siteSlug, redirectTo, dispatch } = this.props;
 		const options = redirectTo ? { redirectTo } : undefined;
-		handleRenewMultiplePurchasesClick( purchases, siteSlug, options );
+		dispatch( handleRenewMultiplePurchasesClick( purchases, siteSlug, options ) );
```

Comment:
This looks off. We should add `handleRenewMultiplePurchasesClick ` to the `mapDispatchToProps` object as a second argument to the `connect()` call below, in order to automatically bind it to `dispatch()` instead, and then invoke it directly here.

Reply:
OK I switched this component to the classic `mapDispatchToProps` approach. It doesn't use the `resolveDomainStatus` function and therefore doesn't need access to the `dispatch` function.